egs/pashto/v1/local/corpus_extraction/words.lst

```diff
@@ -0,0 +1,1862 @@
+خاوره
```

Comment:
this file is too big for github and I don't really see the purpose of it -- could you generate the file automatically?